src/main/ngapp/accio/src/app/editor/editor.component.html

```diff
@@ -1,6 +1,19 @@
-<div id="img-editor">
-  <canvas #canvas appMask [scale]="getScaleFactor()" [originalData]="getImgData()" [scaledData]="getScaledData()" [maskData]="getMaskData()">
-  </canvas>
-  <canvas #hiddenCanvas style="display:none">
-  </canvas>
-</div>
\ No newline at end of file
+<form novalidate [formGroup]="uploadMaskForm" (ngSubmit)="onSubmit()"  
+      *ngIf="display" method="POST" enctype="multipart/form-data" id="mask-form">
+  <mat-form-field appearance="fill">
+    <mat-label>Mask Title</mat-label>
+    <input matInput formControlName="maskName" placeholder="Untitled"/>
```

Comment:
You can write something like "mask name" in placeholder, to label the input when the user hasn't clicked on the field yet.